cpu/kinetis/periph/gpio.c

```diff
@@ -300,7 +304,8 @@ int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
     isr_ctx[ctx_num].cb = cb;
     isr_ctx[ctx_num].arg = arg;
     isr_ctx[ctx_num].state = flank;
-    write_map(port_num(pin), pin_num(pin), ctx_num);
+    /* context map requires ctx_num + 1, as 0 is the no-context value */
```

Comment:
I would have thought `ctx_num < 0` would indicate no context - or how is this meant?

Reply:
Because of the efficient way that someone did the `isr_map`, we need the `isr_map` to store 0 when a pin has no context to map to. This means that when we want to map a pin to `isr_ctx` 0, we actually need to write 1 to `isr_map`.

It would read cleaner to rewrite everything to use -1 for no-context but that isn't compatible with how the bitmapped `isr_map` is done. But I could be mistaken. I didn't try that hard.